plugins/embark-assistant/defs.h

```diff
@@ -162,6 +162,22 @@ namespace embark_assist {
             Major
         };
 
+//        enum class adamantine_ranges : int8_t {
```

Comment:
Are you planning on using this?

Reply:
I have no concrete plans for it. However, it matches the data actually collected for spires, and so I thought it's a good idea to prepare for it while updating the section.